src/css/components/modal.css

```diff
@@ -0,0 +1,71 @@
+.display-none {
+  display: none;
+}
+.display-block {
+  display: block;
+}
+.modal {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  background-color: #00000059;
```

Comment:
Wow, y’all are teaching me all kinds of things in this PR! 😅 When I learned CSS, you couldn’t specify the alpha channel in hex colors, so I learned to use the `rgba()` function instead. So, I never learned that this 8-digit syntax was supported until right now! Thank you!